tests/trainer/test_trainer.py

```diff
@@ -885,7 +888,7 @@ def tokenize_function(examples):
             self.assertLess(max(diff_truth), 0.01, f"Difference {max(diff_truth)} is not within 0.01")
 
             # max diff broken should be very off
-            self.assertGreater(max(diff_broken), 1.5, f"Difference {max(diff_broken)} is not greater than 2")
+            self.assertGreater(max(diff_broken), 1.3, f"Difference {max(diff_broken)} is not greater than 1.3")
```

Comment:
Let's make sure this still passes on GPU

Reply:
used to be `assertGreater(max(diff_broken), 1.5` so `assertGreater(max(diff_broken), 1.3` will pass as well (diff > 1.5 > 1.3)